### PR TITLE
Update navbar title on auth details screen

### PR DESCRIPTION
**Pull request by aidenkeating**
Allow the root nav bar to be updated when the auth details page is shown instead of the auth login page.

https://docs.google.com/presentation/d/1FZL-Wy5D_wUo5ADrfhpRBh-Ny2rGauI3_C8SBrlk6rw/edit#slide=id.g3c52d834c7_0_72

### CODE REVIEW

**Comment by aidenkeating** [comment]
@wei-lee @ciaranRoche Mind taking a look?

File: ios-showcase-template/authentication/AuthenticationRouter.swift [hunk 1]
```diff
@@ -18,17 +18,22 @@ protocol AuthenticationRouter {
 }
 
 class AuthenticationRouterImpl: AuthenticationRouter {
+    public static let PAGE_IDENTITY_PROFILE_TITLE = "Identity Profile"
```
Comment (wei-lee) [inline ios-showcase-template/authentication/AuthenticationRouter.swift]: is this used anywhere?

File: ios-showcase-template/authentication/AuthenticationRouter.swift [hunk 2]
```diff
@@ -18,17 +18,22 @@ protocol AuthenticationRouter {
...
+    init(viewController: AuthenticationViewController, detailsViewController: AuthenticationDetailsViewController, _ options: AuthenticationViewOptions) {
         self.viewController = viewController
         self.detailsViewController = detailsViewController
+        self.options = options
     }
 
     func navigateToUserDetailsView(withIdentify user: User?, andError error: Error?) {
         if let identityInfo = user {
             Logger.info("got user identity: \(identityInfo)")
+            self.options.titleViewController.title = options.authDetailsNavigationTitle
```
Comment (wei-lee) [inline ios-showcase-template/authentication/AuthenticationRouter.swift]: better use `self.options.something` to keep it consistent.

**Review by wei-lee — CHANGES_REQUESTED** [review]
see my comments.

**Review by ciaranRoche — COMMENTED** [review]
Was it decided to remove `Email Verified` from the app? If so might be worth doing it in this PR.

**Comment by aidenkeating** [comment]
@wei-lee Comments addressed.

@ciaranRoche Not sure. @wei-lee Has removing the `Email Verified` (and I guess the `2FA section`) been decided? If so, I'll probably do it in a separate PR.

**Comment by wei-lee** [comment]
@aidenkeating I will let you know as soon as I got confirmation. In the meantime, we can leave it for now and remove it later in a separate PR.

**Review by wei-lee — APPROVED** [review]

**Review by ciaranRoche — APPROVED** [review]
Looks good to merge, keep the other change for a different PR

**Pull request merged**